google-cloud-storage/src/main/java/com/google/cloud/storage/StorageException.java

```diff
@@ -73,4 +82,25 @@ public static StorageException translateAndThrow(RetryHelperException ex) {
     BaseServiceException.translate(ex);
     throw new StorageException(UNKNOWN_CODE, ex.getMessage(), ex.getCause());
   }
+
+  /**
+   * Translate IOException to the StorageException that caused the error. This method defaults to
+   * idempotent always being {@code true}. This method retries the transient issues Connection
```

Comment:
I don't think it retries anything. There's no network connection here.

Reply:
Ah,,, that's not worded well fixing.